### PR TITLE
Build docs using sphinx

### PR DESCRIPTION
**Pull request by liyanhui1228**
The empty .nojekyll file is for letting github pages render the css configurations.

### CODE REVIEW

File: scripts/update_docs.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Build docs
+function build_docs {
+    rm -rf docs/build/
+    make html
+    return $?
+}
+
+build_docs
+
+cp -R docs/build/html/* docs/
```
Comment (duggelz) [inline scripts/update_docs.sh]: Maybe delete `docs/build` after the copy?

File: make.bat [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@ECHO OFF
```
Comment (duggelz) [inline make.bat]: Is this just copied from the quickstart?  I'd rather not include it if we haven't tried it.  We can just say that one must build the docs on Linux, not Windows.

File: scripts/update_docs.sh (hunk 1 above)
Comment (liyanhui1228) [reply scripts/update_docs.sh]: Deleted.

File: make.bat (hunk 2 above)
Comment (liyanhui1228) [reply make.bat]: Yes this is auto generated. Removed.

**Pull request merged**